test/Project.toml

```diff
@@ -2,6 +2,7 @@
 Cbc = "9961bab8-2fa3-5c5a-9d89-47fab24efd76"
 Combinatorics = "861a8166-3701-5b0c-9a16-15d98fcdc6aa"
 ConstraintProgrammingExtensions = "b65d079e-ed98-51d9-b0db-edee61a5c5f8"
+ConstraintSolver = "e0e52ebd-5523-408d-9ca3-7641f1cd1405"
```

Comment:
Normally this shouldn't be needed, right?

Reply:
Yes, I can remove